ncdiff/src/yang/ncdiff/runningconfig.py

```diff
@@ -100,6 +100,9 @@
     (re.compile(r'^ *distribute-list '), 2),
     (re.compile(r'^ *area '), 1),
     (re.compile(r'^ *ip ospf message-digest-key '), 1),
+    (re.compile(r'^ *ospfv3 neighbor '), 1),
+    (re.compile(r'^ *ospfv3 1 ipv(\d) neighbor'), 1),
+    (re.compile(r'^ *ospfv3 1 neighbor '), 1),
```

Comment:
Could this line be:
`    (re.compile(r'^ *ospfv3 \d+ neighbor '), 1),`

Reply:
Updated